helm-charts/example-apiserver/values.yaml

```diff
@@ -97,7 +97,7 @@ tolerations: []
 affinity: {}
 
 apiserver:
-  create: false
+  create: true
```

Comment:
This creates the APIServer outside of konk, so what's the point? We want to prove it works inside konk.